backend/api/organisations/resources.py

```diff
@@ -315,18 +315,6 @@ def get(self):
         except Exception:
             manager_user_id = None
 
-        if manager_user_id is not None:
```

Comment:
@ramyaragupathy I removed that restriction in order to be able to query the organizations of another user. Is that ok?